packages/drivers/odsp-driver/src/test/odspError.spec.ts

```diff
@@ -30,6 +31,26 @@ describe("Odsp Error", () => {
 		},
 	} as Response;
 
+	/**
+	 * Checks is the networkError complies with the {@link IGenericNetworkError}.
+	 */
+	function isIGenericNetworkError(input: any): input is IGenericNetworkError {
+		return (
+			typeof input.errorType === typeof DriverErrorTypes.genericNetworkError &&
```

Comment:
Could this just compare `input.errorType` to `DriverErrorTypes.genericNetworkError`, without `typeof`? Same for the other helper function.